File: Aula 4/Exercicio11/app/src/main/java/br/edu/decision/exercicio11/MainActivity.java
package br.edu.decision.exercicio11;

import android.os.Handler;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {
    TextView textView;
    private String texto = null;
    private Handler handler = new Handler();

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        textView = (TextView) findViewById(R.id.textView);
        new Thread(){
                public void run(){
                    texto = getResources().getString(R.string.texto);
                    handler.post(atualizarTextView);
                }
        }.start();
    }

    private Runnable atualizarTextView = new Runnable() {
        @Override
        public void run() {
            textView.setText(texto);
        }
    };
}

File: Aula_2/Exercicio3/app/src/main/java/infnet/edu/br/exercicio3/MainActivity.kt
package infnet.edu.br.exercicio3

import android.content.Intent
import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import android.widget.TextView
import infnet.edu.br.exercicio3.cadastro.CadastroActivity
import infnet.edu.br.exercicio3.cadastro.DetalhesUsuarioActivity
import infnet.edu.br.exercicio3.contar.ContarActivity

class MainActivity : AppCompatActivity() {

    var contar = 0;
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)


        var btContar = findViewById<Button>(R.id.btContar)

        btContar.setOnClickListener {

            val intent = Intent(this, ContarActivity::class.java)
            startActivity(intent)

        }

        var btCad = findViewById<Button>(R.id.btCadastro)
        btCad.setOnClickListener {

            val intent = Intent(this, CadastroActivity::class.java)
            startActivity(intent)

        }
    }



}
